### PR TITLE
B906: ignore functions whose _fields attribute can't contain ast.AST subnodes

### LINKED ISSUE #330: B906: Do not warn for missing `visit` calls for bottom-level nodes
Hey hey!

Just upgraded to the latest version to find some neat warnings (B906) about `ast.NodeVisitor` functions missing a further visit call. The suggestion was to call `generic_visit`. This is awesome in general, but unnecessary for:

- Global
- Nonlocal
- Name
- MatchAs
- alias

Might be others as well, but these were the ones I hit. If you think it's reasonable, I'd like to suggest that bugbear suppresses the warnings for these nodes.

byeeeee

### PR DESCRIPTION
**Pull request by jakkdl**
fixes #330

### CODE REVIEW

**Review by felix-hilden — COMMENTED** [review]
Thanks for working on this so quickly 🙏 looking good apart from one thing below.

**Comment by cooperlees** [comment]
Will leave this for another release :)

File: bugbear.py [hunk 1]
```diff
@@ -997,12 +997,30 @@ def check_for_b906(self, node: ast.FunctionDef):
...
+            # iterated over in ast.NodeVisitor.generic_visit
+            or not getattr(class_type, "_fields", None)
+            # or can't contain any ast subnodes that could be visited
+            # See https://docs.python.org/3/library/ast.html#abstract-grammar
+            or class_type.__name__
+            in (
+                "alias",
+                "Constant",
+                "Global",
+                "ImportFrom",
```
[the remark this answers is not in the document]
Comment (jakkdl) [reply bugbear.py]: oops, yep. Thanks!

**Comment by jakkdl** [comment]
Rebased on top of main and removed `ImportFrom`

**Review by cooperlees — APPROVED** [review]
Sound research, tests show it does what we want and if @felix-hilden is happy I am happy here.

**Pull request merged**